docs/firestore-bundle-builder/api-reference.md

```diff
@@ -90,7 +90,7 @@ type QueryCondition = {
       | "not-in"
```

Comment:
im not sure why we have docs for this extension here? We should remove irrelevant docs probably? @pr-Mais